web/components/inbox/content/inbox-issue-header.tsx

```diff
@@ -60,11 +60,11 @@ export const InboxIssueActionsHeader: FC<TInboxIssueActionsHeader> = observer((p
   const issue = inboxIssue?.issue;
   // derived values
   const isAllowed = !!currentProjectRole && currentProjectRole >= EUserProjectRoles.MEMBER;
-  const canMarkAsDuplicate = isAllowed && inboxIssue?.status === -2;
+  const canMarkAsDuplicate = isAllowed && (inboxIssue?.status === 0 || inboxIssue?.status === -2);
```

Comment:
Not a blocker for this PR, but please try to use Enums for inbox issue status instead of numbers